src/libmongoc/tests/unified/runner.c

```diff
@@ -79,6 +79,16 @@ skipped_unified_test_t SKIPPED_TESTS[] = {
    {"cursors are correctly pinned to connections for load-balanced clusters", "listIndexes pins the cursor to a connection"},
    /* libmongoc does not pin connections to cursors. It cannot force an error from waitQueueTimeoutMS by creating cursors in load balanced mode. */
    {"wait queue timeout errors include details about checked out connections", SKIP_ALL_TESTS},
+   /* CDRIVER-4277: Change streams support for user-facing PIT pre- and post-images */
+   {"change-streams-pre_and_post_images", SKIP_ALL_TESTS},
+   /* CDRIVER-3973, CDRIVER-4305, CDRIVER-4279, CDRIVER-4321: unified change stream tests */
+   {"change-streams", SKIP_ALL_TESTS},
+   {"change-streams-errors", SKIP_ALL_TESTS},
+   {"change-streams-resume-allowlist", SKIP_ALL_TESTS},
+   {"change-streams-resume-errorlabels", SKIP_ALL_TESTS},
+   /* CDRIVER-4309: Use the count command instead of collStats to implement estimatedDocumentCount */
```

Comment:
I originally included this change in the same PR but decided to pull it out. In any event, the code is ready to go (mostly just some deletions) so I'll plan to take this ticket on immediately after we merge this.

Reply:
[CDRIVER-4309](https://jira.mongodb.org/browse/CDRIVER-4309) was merged in #984 and I've since rebased this PR on master.